### PR TITLE
Chore: Cleanup unused versionError from ContentSidebar/DetailsSidebar

### PR DESCRIPTION
**Pull request by pramodsum**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @pramodsum has signed the CLA. Thanks for the pull request!

File: src/components/ContentSidebar/ActivityFeed/activity-feed/ActiveState.js [hunk 1]
```diff
@@ -93,16 +93,6 @@ const ActiveState = ({
                             )}
                         </li>
                     );
-                case 'file_version_error':
```
Comment (pramodsum) [inline src/components/ContentSidebar/ActivityFeed/activity-feed/ActiveState.js]: This error no longer gets returned from the File Versions API

File: src/components/ContentSidebar/ActivityFeed/version/VersionError.js [hunk 2]
```diff
@@ -1,32 +0,0 @@
...
- */
-
-import * as React from 'react';
-import { FormattedMessage } from 'react-intl';
-
-import messages from '../../../messages';
-
-import './Version.scss';
-
-function getErrorMessage(errorCode: string): React.Node {
```
Comment (pramodsum) [inline src/components/ContentSidebar/ActivityFeed/version/VersionError.js]: This component was specifically to display the 'tooManyVersions' error which no longer gets returned from the API

File: src/components/ContentSidebar/ContentSidebar.js [hunk 3]
```diff
@@ -351,12 +349,6 @@ class ContentSidebar extends PureComponent<Props, State> {
                 total_count: 0,
                 entries: []
             },
-            versionError: {
```
Comment (pramodsum) [inline src/components/ContentSidebar/ContentSidebar.js]: Not used by DetailsSidebar or ActivitySidebar

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #429

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

**Pull request merged**